### PR TITLE
fix: version parsing and project name

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [X] Ready for review
- [X] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Fix 2 issues
1. When `sbt.version` in `project/build.properties` containing spaces around `=` sign failing to use new plugin (e.g. `sbt.version = 1.2.8`) and fall back to legacy
2. For multi-project projects returning back package name `.`. Fix it so it will appear nicely in UI.

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -117,6 +117,8 @@ async function pluginInspect(root: string, targetFile: string, options: any): Pr
     const sbtArgs = buildArgs(options.args, false, true);
     const sbtVersion = getSbtVersion(root, targetFile);
     const sbtPluginPath = generateSbtPluginPath(sbtVersion);
+    const packageName = path.basename(root);
```
Comment (lili2311) [inline lib/index.ts]: is this what we do for legacy too?

Comment (dkontorovskyy) [reply lib/index.ts]: yes

File: lib/parse-sbt.ts [hunk 2]
```diff
@@ -204,8 +204,8 @@ function parseSbtPluginResults(sbtOutput: string): DepTree {
   }
 
   const depTree = {
-    name: '.',
-    version: '1.0.0',
+    name: packageName,
```
Comment (lili2311) [inline lib/parse-sbt.ts]: ❤️

File: lib/index.ts (hunk 1 above)
Comment (dkontorovskyy) [reply lib/index.ts]: https://github.com/snyk/snyk-sbt-plugin/blob/4c042a26981a424a12e8db4c3dc5c003e18d9a2a/lib/index.ts#L35

**Pull request merged**